src/pyodide/BUILD.bazel

```diff
@@ -44,21 +55,17 @@ capnp_embed(
     deps = ["pyodide_extra_capnp_file"],
 )
 
-capnp_embed(
-    name = "pyodide_lock_file_embed",
-    src = "generated/pyodide-lock.json",
-    deps = ["pyodide-lock.js@rule"],
-)
-
 cc_capnp_library(
     name = "pyodide_extra_capnp",
     srcs = ["generated/pyodide_extra.capnp"],
     visibility = ["//visibility:public"],
     deps = [
         ":pyodide_extra_file_embed",
-        ":pyodide_lock_file_embed",
         ":pyodide_packages_archive_embed",
         ":python_entrypoint_file_embed",
+    ] + [
+        ":pyodide_lock_" + package_date + "_file_embed"
+        for package_date in PYTHON_LOCKFILES.keys()
```

Comment:
Why don't we make a lockfiles `filegroup`?

Reply:
What's the advantage of doing this?